core/pipeline/label.go

```diff
@@ -82,19 +82,24 @@ func LoadGitLabels(workdir string) ([]Label, error) {
 		return nil, err
 	}
 
+	labels := []Label{}
+
 	origin, err := repo.Remote("origin")
-	if err != nil {
-		return nil, err
-	}
+	if err == nil {
+		urls := origin.Config().URLs
```

Comment:
just checking that there couldn't be a case where both `err` and `origin` are null here. Wondering if we should safely check for `origin != null` in the if statement above